packages/babel-preset-env/test/utils.spec.js

```diff
@@ -56,6 +56,8 @@ describe("utils", () => {
       const options = ["one", "two", "three"];
       expect(findSuggestion(options, "onr")).toEqual("one");
       expect(findSuggestion(options, "tree")).toEqual("three");
+      expect(findSuggestion(options, "")).toEqual("one");
+      expect(findSuggestion(options, "xxx")).toEqual("one");
```

Comment:
Is this a behaviour change?

Reply:
Assuming a set of options is passed, it would always return a string.